### ISSUE TITLE
documentation assumes you know what pug is and what it's for

### ISSUE THREAD

**Issue by LegalizeAdulthood** [pugjs/pug#2538]
OK, I go on vacation for a week and when I come back, I'm searching for the jade reference and I keep getting a page on 'pug' that looks completely different and doesn't say what 'pug' is for or what it does at the top of it's page.

I now know that the people at jade software are insisting that you divorce yourself completely from using the name 'jade', but for folks like me your top-level page must at least say the following clearly at the top of the page:

- pug is a template rendering engine for node
- pug was forced to change name to 'pug' due to a legal claim

I kept looking for jade-lang.com and kept getting redirected to a page on 'pug' that didn't clearly say what pug was and so I just assumed that someone had hijacked the domain and was squatting over the place I really wanted to see.  (And sorry, but your color scheme is kinda ugly and it just sent me leaving for something else as this didn't appear to be what I was looking for.)

Your opening paragraph:

> The general rendering process of Pug is simple. pug.compile() will compile the Pug source code into a JavaScript function that takes a data object (called “locals”) as argument. Call that resultant function with your data, and voilà! it will return a string of HTML rendered with your data.

The core responsibility of pug is buried through implication in that paragraph.  Rendering is mentioned, but only by inference can I figure out that pug is the thing that I used to know as jade.  Seriously.  Maybe I'm just thick or maybe the author of the docs is too close to the code to look at it as a stranger, but it wasn't until I googled "jade pug" that I found a thread on reddit discussing the forced name change.

Adding the two bullet points above will make it clear to folks like me who are looking for jade that they are indeed in the right place and the name was changed to protect the vanity of idiots.

**Comment by robbyoconnor** [comment]
This has been the case for awhile now... it's not like pug just started last week...

**Comment by TimothyGu** [comment]
Our website is newly written, and we haven't written a landing page yet. Help welcome.

**Comment by ForbesLindesay** [comment]
Yes, the landing page should have a really big banner telling you that it was formerly known as jade and linking to the migration guide.

**Comment by cekvenich** [comment]
TimothyGu I want to help on anything, including documentation.
But NO ONE will write documentation w/o a being able to draft. For example in Wiki.

So to resolve the stand of, I got pug.host domain and will install MediaWiki on it. 
I hope other's help.
Once a 'article' is discussed by community - it will be pushed to doc in git.

**Comment by TimothyGu** [comment]
> Once a 'article' is discussed by community - it will be pushed to doc in git.

Why don't you put it in Git now, so you won't have to transfer the doc later?

We can discuss the doc in a pull request.

**Comment by robbyoconnor** [comment]
:+1: to this!

**Comment by ForbesLindesay** [comment]
You can draft in git.  Simply fork the repository and your fork will act as your own personal draft, visible to everyone.  When you submit a pull request, we can use GitHub's commenting features to give you feedback if need be, and simply click the "merge" button, once it's ready.
